Fitness/Controllers/HomeController.swift

```diff
@@ -89,61 +94,83 @@ class HomeController: UIViewController {
         statusBarBackgroundColor = UIView(frame: CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: 21))
```

Comment:
Bring constant 21 out.

Reply:
Looking into this, this view doesn't actually do anything--the view it sits on is already white. Going to just remove it!